src/data/stripe.ts

```diff
@@ -0,0 +1,80 @@
+import Stripe from 'stripe';
+const { MODE, STRIPE_PK, STRIPE_SK } = import.meta.env;
```

Comment:
🔍 `import.meta.env` is the new platform-standard way to get things we typically get from `process.env`. It's how Vite (which powers Astro) exposes env variables at build time. 

- [`import.meta.env` on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import.meta)

Reply:
👍